var/spack/repos/builtin/packages/pango/package.py

```diff
@@ -19,6 +19,7 @@ class Pango(MesonPackage):
 
     license("LGPL-2.1-or-later")
 
+    version("1.90.0", sha256="f138f964a8390c8aecc6c79dc18756989cc2d4db624b12f94a01a92ec57111ff")
```

Comment:
Avoid this one. It's an unstable development release leading towards Pango 2.0, which will be major new version, and will not be API or ABI compatible with Pango 1.x. [https://download.gnome.org/sources/pango/1.90/pango-1.90.0.news]

Having 1.90 will also prevent updates to the 1.x series that are more important, https://download.gnome.org/sources/pango/1.52/.

Reply:
I've instead pushed pango to v1.52.2 in this PR following with the 1.x series that was linked.

Reply:
I didn't check, but do you want to add a comment to prevent this from happening again?